app/javascript/flavours/glitch/packs/admin.js

```diff
@@ -1,6 +1,27 @@
 import 'packs/public-path';
+import loadPolyfills from 'flavours/glitch/util/load_polyfills';
 import ready from 'flavours/glitch/util/ready';
 
+function setupSidebar() {
```

Comment:
Maybe rename it to `main` and move the `ready` call from below there?

Reply:
Move it to the inside of ready function?

Reply:
No, the opposite: have `ready` be called by this (`setupSidebar`, which I propose should be renamed to `main`) function after loading polyfills, as other packs do.

Reply:
OK, changed with 41396a7